pages/learn/accounts/account.md

```diff
@@ -56,60 +56,8 @@ API keys can also be generated using the API, [CLI][cli-keys], and [Node.js][nod
 
 ## Fleet members
 
-When a fleet needs to be shared with more than one user, the fleet owner can add new members. With paid accounts, it's possible to assign a level of access for a new member, based on the following types:
+When a fleet needs to be shared with more than one user, the fleet owner can add new members. Checkout out [team management][team] in BalenaCloud to understand member types, how to add a member and more.
```

Comment:
`Checkout out`?

Should it be `Check out` or `Checkout our`?

Reply:
Also the preview shows this text as `Checkout out [team management][team] in BalenaCloud to understand member types, how to add a member and more.` rather than the proper text for `[team management][team]`, see: https://8c488a9d.balenacloud-docs.pages.dev/learn/accounts/account/#fleet-members

Reply:
Yes, the latest commit fixed that; I saw that this didn't have a link, which I added in the latest amend.